eth_verifier/test/Deserialize.t.sol

```diff
@@ -234,29 +234,57 @@ contract DeserializeTest is Test {
+        Scalar.FE[] memory public_inputs = MsgPk.deser_public_inputs(public_inputs_serialized);
+        assertEq(Scalar.FE.unwrap(public_inputs[2]), 0x000000000000000000000000000000000000000000002149A7476FD365F3E060);
+    }
+
     // WARN: this doesn't assert anything, it only executes this deserialization
     // for gas profiling.
     function test_deserialize_linearization_profiling_only() public {
-        bytes memory linearization_serialized = vm.readFileBinary("unit_test_data/linearization.mpk");
+        bytes memory linearization_serialized = vm.readFileBinary("linearization.mpk");
         MsgPk.deser_linearization(MsgPk.new_stream(linearization_serialized), index);
     }
 
+    // WARN: this doesn't assert anything, it only executes this deserialization
+    // for gas profiling.
+    function test_decode_linearization_profiling_only() public {
+        bytes memory linearization_rlp = vm.readFileBinary("linearization.rlp");
+        abi.decode(linearization_rlp, (Linearization));
+    }
+
     // WARN: this doesn't assert anything, it only executes this deserialization
     // for gas profiling.
     function test_deserialize_lagrange_bases_profiling_only() public {
-        bytes memory lagrange_bases_serialized = vm.readFileBinary("unit_test_data/lagrange_bases.mpk");
+        bytes memory lagrange_bases_serialized = vm.readFileBinary("lagrange_bases.mpk");
         MsgPk.deser_lagrange_bases(lagrange_bases_serialized);
     }
 
-    function test_deserialize_scalar() public {
-        bytes memory scalar_serialized = hex"550028f667d034768ec0a14ac5f5a24bbcad7117110fc65b7529e003a0708419";
-        Scalar.FE scalar = MsgPk.deser_scalar(scalar_serialized);
-        assertEq(Scalar.FE.unwrap(scalar), 0x198470A003E029755BC60F111771ADBC4BA2F5C54AA1C08E7634D067F6280055);
+    // WARN: this doesn't assert anything, it only executes this deserialization
+    // for gas profiling.
+    function test_deserialize_verifier_index_profiling_only() public {
+        bytes memory verifier_index_serialized = vm.readFileBinary("verifier_index.mpk");
+        MsgPk.deser_verifier_index(MsgPk.new_stream(verifier_index_serialized), index);
     }
 
-    function test_deserialize_public_input() public {
-        public_inputs_serialized = vm.readFileBinary("public_inputs.mpk");
-        Scalar.FE[] memory public_inputs = MsgPk.deser_public_inputs(public_inputs_serialized);
-        assertEq(Scalar.FE.unwrap(public_inputs[2]), 0x000000000000000000000000000000000000000000002149A7476FD365F3E060);
+    // WARN: this doesn't assert anything, it only executes this deserialization
+    // for gas profiling.
+    function test_deserialize_prover_proof_profiling_only() public {
+        bytes memory prover_proof_serialized = vm.readFileBinary("prover_proof.mpk");
+        MsgPk.deser_prover_proof(MsgPk.new_stream(prover_proof_serialized), prover_proof);
+    }
+
+    // WARN: this doesn't assert anything, it only executes this deserialization
```

Comment:
```suggestion
    // NOTICE: this doesn't assert anything, it only executes this deserialization
```

Reply:
changed them to `INFO:` because `NOTICE:` doesn't gets highlighted in my editor :neckbeard: